### PR TITLE
Feat/utils and constants

### PR DESCRIPTION
**Pull request by krishnaagarwal1506**
5 Main Changes:-
1. utils file
2. constant file
3. App.jsx
4. webpack.config.js
5. .gitignore

and rest of the file are assets for Homepage
In the constant file the variable names are not capitalize as this is the part of the old PR that I am dividing into smaller PRs
the constant files is already updated and approved

### CODE REVIEW

File: public/main.js
Comment (shelly19gkmit) [inline public/main.js:1]: Why is this committed on git?

File: src/constant/index.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
+//facility compoenent data
+export const facilitesData = [
```
Comment (shelly19gkmit) [inline src/constant/index.js:2]: facilitesData itself tells what that means, you can remove the comment

File: src/constant/index.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  {
+    imgSrc: "drinking-water.webp",
+    heading: "Drinking Water",
+    content:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dicta, fugiat mollitia.",
+  },
+];
+
+//testimonials component data
+export const testimonialsData = [
```
Comment (shelly19gkmit) [inline src/constant/index.js:48]: same as above

File: src/utils/getImageArray.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function importAll(requireContext) {
+  let images = {};
+  requireContext.keys().map((item, index) => {
+    images[item.replace("./", "")] = requireContext(item);
+  });
+  return images;
+}
+
+export { importAll };
```
Comment (shelly19gkmit) [inline src/utils/getImageArray.js:9]: Refactor this and come up with an approach

Comment (krishnaagarwal1506) [reply src/utils/getImageArray.js:9]: Can you give me any idea on how to refractor this.

**Pull request closed without merging**